tests/unit/10_build-env.bats

```diff
@@ -44,6 +44,20 @@ teardown() {
   assert_success
 }
 
+@test "(build-env) buildpack if Dockerfile present" {
```

Comment:
If I understand this correctly, the test above will also be a success if Dokku builds `$TEST_APP` using `Dockerfile`, ignoring the fact that it should build using the buildpack set in `BUILDPACK_URL` instead. 

I'd be glad if you could rebase this as I'm pretty much slow to respond here due to a deadline :)